submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+              </label>
+              <input
+                type="radio"
+                name="tone"
+                id="formal"
+                class="visually__hidden"
+              />
+              <label for="formal" class="button form-select_tone_radio_label">
+                <span class="form-select_tone_icon formal">Media/formal</span>
+              </label>
+            </section>
+            <section class="form-range_urgent">
+              <span
+                class="form-sections_headers_text form-sections_headers_text_urgent"
+              >
+                Tell us how urgent this job is
+              </span>
+              <div class="wrapper_form-range">
+                <input
+                  type="range"
+                  name="urgent"
+                  id=""
+                  min="0"
+                  max="2"
+                  step="1"
+                  value="1"
+                  class="form-range_input"
+                />
+                <div class="range_text">
+                  <span>I got time</span>
+                  <span class="orange">Average</span>
+                  <span>Yesterday</span>
+                </div>
+              </div>
+              <aside class="form-urgent_info">
+                <span class="form-urgent_info_header">Timeline info</span>
+                <p class="form-urgent_info_text">
+                  Lorem ipsum dolor sit <span class="orange">Orange</span>.Lorem
+                  ipsum dolor sit amet.Lorem ipsum dolor sit amet.
+                </p>
+              </aside>
+            </section>
+          </div>
+          <section class="form-sign_pay_method">
+            <span
+              class="form-sections_headers_text form-sign_pay_method_header"
+            >
+              Sign up and choose a payment method
+            </span>
+
+            <span class="form-sign_pay_method_link"
+              >Already have an account with us ?<a
+                href="#"
+                class="form-sign_pay_method_link"
+                >Sign in</a
+              ></span
+            >
+            <input
+              type="text"
+              id="input_enter_name"
+              name=""
+              class="form-sign_pay_method_inputs input_enter_name"
+              placeholder="Enter your name"
+              minlength="5"
+              maxlength="20"
+              pattern="^[a-zA-Z]+$"
+              required
+            />
+            <span class="input_name title"></span>
+            <div class="wrapper_form-email_password">
+              <input
+                type="email"
+                placeholder="example@domaine.com"
+                name=""
+                class="form-sign_pay_method_inputs input_email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                required
+              />
+              <span class="input_name input_email_title"></span>
```

Comment:
For this text use `<label>`, avoid fill content using pseudoelements.

Reply:
Done.